conans/model/version.py

```diff
@@ -128,31 +127,19 @@ def main(self):
 
     @property
     def major(self):
-        try:
-            return self.main[0]
-        except IndexError:
-            return None
+        return self.main[0] if len(self.main) > 0 else None
```

Comment:
The try-except is actually very idiomatic python, better leave it. 
The rational is "ask for forgiveness is better than ask for permission". If you expect that a major version will be defined most of the time (sounds reasonable), then the try-except is more idiomatic than checking ``if`` exist the element

Reply:
Ok, thanks for the clarification, I'll revert those changes.